### PR TITLE
Add meta description and seo title for register pages

### PR DESCRIPTION
**Pull request by dankmitchell**
### Context
We want to add a Google friendly seo title and meta description

### Changes proposed in this pull request
Add additional fields into the admin to enable the ability of an seo title and meta description

### Guidance to review
`/admin` edit any register to add an seo title and meta description

### CODE REVIEW

File: app/views/registers/show.html.haml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-- @page_title = "#{@register.register_name} register"
-- @page_description = @register.register_description
+- @page_title = "#{@register.seo_title.present? ? @register.seo_title : @register.register_name + ' register'}"
```
Comment (gidsg) [inline app/views/registers/show.html.haml]: I think you could express this more concisely as `[@register.seo_title, @register.register_name].find(&:present?)`

File: app/views/registers/show.html.haml [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
+- @page_description = "#{@register.meta_description.present? ? @register.meta_description : @register.register_description}"
```
Comment (gidsg) [inline app/views/registers/show.html.haml]: I think we should also consider using this for the secondary heading span within the H1 as I don't think meta description alone will have a big SEO effect.

Comment (dankmitchell) [reply app/views/registers/show.html.haml]: The meta description is for google search only.

File: app/views/registers/show.html.haml [hunk 3]
```diff
@@ -1,5 +1,5 @@
-- @page_title = "#{@register.seo_title.present? ? @register.seo_title : @register.register_name + ' register'}"
-- @page_description = "#{@register.meta_description.present? ? @register.meta_description : @register.register_description}"
+- @page_title = "#{[@register.seo_title, @register.register_name].find(&:present?)}"
```
Comment (gidsg) [inline app/views/registers/show.html.haml]: do you still want to append `register` on the end? Or only for the `register_name` maybe?

Comment (dankmitchell) [reply app/views/registers/show.html.haml]: Yes, you are right.

**Pull request merged**